cf-deploy-teardown/tasks/k8s-ready-state-check.sh

```diff
@@ -0,0 +1,59 @@
+#Script to determine is the K8s host is "ready" for cf deployment
+
+set -ex
+
+# cgroup memory & swap accounting in /proc/cmdline
+
+cat /proc/cmdline | grep -w "cgroup_enable=memory"
```

Comment:
`cat` isn't needed here or in the next check: `grep -w "cgroup_enable=memory" /proc/cmdline` instead.